src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -334,22 +334,13 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
             serializer.Serialize(writer, ws.Nodes);
 
             //notes
-           writer.WritePropertyName("Notes");
+            writer.WritePropertyName("Notes");
```

Comment:
@gregmarr  My preference is that we would merge `Note` into `Annotation`. A note, with no associated nodes would just have an empty node collection. That will require back-end work by the Dynamo team, which I am not sure they have in their backlog. @Racel?